pudb/source_view.py

```diff
@@ -222,3 +278,51 @@ def shipout_line():
                 PythonLexer(stripnl=False), UrwidFormatter())
 
         return result
+
+from enum import Enum
+class ParseState(Enum):
```

Comment:
Agreed!  I now have a 2.6 environment setup and have tested removing the enum with 2.6 & 3.5 (works).  Will push in a moment.